ansible/example_vars/cluster-config.yaml

```diff
@@ -44,7 +44,7 @@ cluster_nodes:
   - name: cp-1
     type: control-plane
     vm:
-      disk_size: 60
+      disk_size: 120
```

Comment:
Haha...that makes sense...probably makes it easier to use out of the box with the RH hosted AI Svc vs a self-hosted AI ensemble...smart.